src/odemis/gui/conf/file.py

```diff
@@ -119,9 +122,11 @@ def set_many(self, section, option_value_list):
         self.write()
 
     def get(self, section, option):
-        """ Get the value of an option """
+        """ Get the value of an option
+        returns (unicode): the value is converted from UTF-8
+        """
         try:
-            return self.config.get(section, option)
+            return self.config.get(section, option).decode("utf-8", "replace")
```

Comment:
maybe add: U+FFFD � REPLACEMENT CHARACTER characters will be inserted for any bytes that can't be decoded